README.md

```diff
@@ -126,47 +88,84 @@ kustomize build 'github.com/kubernetes-sigs/descheduler/kubernetes/deployment?re
+| `maxNoOfPodsToEvictPerNode` | `nil` | maximum number of pods evicted from each node (summed through all strategies) |
+| `maxNoOfPodsToEvictPerNamespace` | `nil` | maximum number of pods evicted from each namespace (summed through all strategies) |
+
+### Evictor Plugion configuration (Default Evictor)
+
+The Default Evictor Plugin is used by default for filtering pods before processing them in an eviction strategy plugin, or for applying a PreEvictionFilter of pods before eviction. You can also create your own Evictor Plugin or use the Default one provided by Descheduler.  Other uses for the Evictor plugin can be to sort, filter, validate or group pods by different criteria, and that's why this is handled by a plugin and not configured in the top level config.
 
-The policy includes a common configuration that applies to all the strategies:
 | Name | Default Value | Description |
 |------|---------------|-------------|
 | `nodeSelector` | `nil` | limiting the nodes which are processed |
 | `evictLocalStoragePods` | `false` | allows eviction of pods with local storage |
 | `evictSystemCriticalPods` | `false` | [Warning: Will evict Kubernetes system pods] allows eviction of pods with any priority, including system pods like kube-dns |
 | `ignorePvcPods` | `false` | set whether PVC pods should be evicted or ignored |
-| `maxNoOfPodsToEvictPerNode` | `nil` | maximum number of pods evicted from each node (summed through all strategies) |
-| `maxNoOfPodsToEvictPerNamespace` | `nil` | maximum number of pods evicted from each namespace (summed through all strategies) |
 | `evictFailedBarePods` | `false` | allow eviction of pods without owner references and in failed phase |
+|`labelSelector`|(see [label filtering](#label-filtering))|
+|`priorityThreshold`|int (see [priority filtering](#priority-filtering))|
+|`thresholdPriorityClassName`|string (see [priority filtering](#priority-filtering))|
+|`nodeFit`|bool (see [node fit filtering](#node-fit-filtering))|
+
+### Example policy
+
+As part of the policy, you will start deciding which top level configuration to use, then which Evictor plugin to use (if you have your own, the Default Evictor if not), followed by deciding the configuration passed to the Evictor Plugin. After that you will enable/disable eviction strategies plugins and configure them properly.
 
-As part of the policy, the parameters associated with each strategy can be configured.
-See each strategy for details on available parameters.
+See each eviction strategy plugin section for details on available parameters.
 
 **Policy:**
 
 ```yaml
-apiVersion: "descheduler/v1alpha1"
+apiVersion: "descheduler/v1alpha2"
 kind: "DeschedulerPolicy"
-nodeSelector: prod=dev
-evictFailedBarePods: false
-evictLocalStoragePods: true
-evictSystemCriticalPods: true
-maxNoOfPodsToEvictPerNode: 40
-ignorePvcPods: false
-strategies:
-  ...
+nodeSelector: "node1" # you don't need to set this, if not set all will be processed
```

Comment:
I think the smallest valid selector is in a `key=value` form. E.g. `node=node1`

Reply:
:heavy_check_mark: